.devcontainer/devcontainer.json

```diff
@@ -36,7 +36,7 @@
   },
   // Comment out connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
   "remoteUser": "vscode",
-  "hostRequirements": {
-    "cpus": 8
-  }
+	"hostRequirements": {
+		"cpus": 4,
```

Comment:
This change is unrelated to this PR. Please undo except there are good reasons for that change (even then it should become a distinct PR). Thanks.